docs/finality-provider-operation.md

```diff
@@ -810,48 +786,38 @@ This command should ask to
 `confirm transaction before signing and broadcasting [y/N]:` and output the
 transaction hash.
 
-### 6.2.4. Refund Finality Provider
-
-To support the gas costs associated with committing randomness, which are not
-refunded by the protocol, we recommend setting up a refunding flow.
-This involves periodically transferring rewards from a beneficiary key to the
-operational key used by the finality provider.
+### 6.3. Set Up Operation Key
 
-#### How to set up the refunding process
+Finality providers consume gas for operations with Babylon Genesis.
+Given that the cost of sending finality signatures is refunded automatically
+after success, the only cost of operating a finality provider is randomness
+commit, which is made periodically with very low gas cost.
 
-1. Ensure you have two keys:
-   * Beneficiary Key: Receives finality provider rewards.
-   * Operational Key: Used by the finality provider daemon to submit transactions.
+Therefore, it is recommended to use a separate key specifically for operations.
+This leaves key used for registration key isolated and only needed for claiming
+rewards. Given that the operation key is totally replaceable, it can hold only
```

Comment:
How is it replaceable?

Reply:
Any key can be the operation key as long as it's funded. It's not bonded to the fp. Will rephrase as following:

```
The operation key is totally replaceable in the sense that it is not
tied to a finality provider, and it can be replaced by any key that is properly
funded.
```